package.json

```diff
@@ -17,7 +17,11 @@
     "test:integration": "pnpm -r --filter=./integration-tests/* run test",
     "test:types": "pnpm -r --filter=./packages/* run test:types",
     "test": "pnpm -r --filter=./packages/* run test",
-    "typesync": "pnpm exec typesync && pnpm -r exec typesync && pnpm install"
+    "typesync": "pnpm exec typesync && pnpm -r exec typesync && pnpm install",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint src",
+    "test:lint": "npm run lint",
+    "test:format": "npm run format"
```

Comment:
This is just an alias for format, which isn't very helpful.

I'd like a `test:format` to run prettier and exit with code 1 if there are any changes

There is a `prettier --check` command which looks like the right sort of thing to me